kubectl-plugin/test/e2e/testdata/ray-job.interactive-mode.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: ray.io/v1
+kind: RayJob
+metadata:
+  name: rayjob-sample
+spec:
+  # The current value is "InteractiveMode", meaning that it will wait for user to submit job and provide the job submission ID
+  submissionMode: 'InteractiveMode'
+  runtimeEnvYAML: |
```

Comment:
@kevin85421 @MortalHappiness what's the expected behavior with runtime env when using `InteractiveMode`. It should be ignored right?